doc/user/CONTRIBUTING.md

```diff
@@ -0,0 +1,55 @@
+This doc is a WIP. There is more useful information in the [README](./README.md).
+
+# Set up
+
+Rather than work on `github.com/materializeinc/materialize` itself, we recommend
+working on your own fork of Materialize; this makes branches much tidier.
+branches tidier.
+
+1. Fork `github.com/materializeinc/materialize`. You can do this by going to
+   that address and clicking **Fork**.
```

Comment:
this could plausibly be a link instead of preventing linking by putting it in code?

```suggestion
1. Fork <https://github.com/materializeinc/materialize>. You can do this by going to
   that address and clicking **Fork**.
```